### PR TITLE
[Chart] Adds backup rentention rule for WAL files

### PR DESCRIPTION
**Pull request by John2020-cyber**
Added a global field in postgrescluster.yaml to limit WAL file size to 5G and file Count to 1

### CODE REVIEW

**Comment by stakater-user** [comment]
@John2020-cyber Validation Successful

File: stakater/postgres-instance/templates/postgrescluster.yaml [hunk 1]
```diff
@@ -26,9 +26,15 @@ spec:
             resources:
               requests:
                 storage: {{ .Values.postgres.repo.storage }}
+#--------Added to reduce WAL file size and WAL file count-----------#
```
Comment (usamaahmadkhan) [inline stakater/postgres-instance/templates/postgrescluster.yaml:29]: Why hard code in the template directly?
They are not configurable in values.yaml?

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: stakater/postgres-instance/templates/postgrescluster.yaml (hunk 1 above)
Comment (John2020-cyber) [reply stakater/postgres-instance/templates/postgrescluster.yaml:29]: I tried lint, it was giving some errors. Will try again and push it after that again.

Comment (John2020-cyber) [reply stakater/postgres-instance/templates/postgrescluster.yaml:29]: Fixed that. It works now

**Review by usamaahmadkhan — APPROVED** [review]

**Pull request merged**